src/fragments/start/getting-started/vue/setup.mdx

```diff
@@ -132,24 +132,7 @@ When working with a Vue `create-app` that uses [Vite](https://vitejs.dev) you mu
 </body>
 ```
 
-**2.** Update the `vite.config.ts` and add in a resolve object inside the `defineConfig({})` as seen below.
```

Comment:
are you sure of this @renebrandel ? I thought this was a limitation with Vite, so I find it weird it is resolved.